src/components/CreateProductButton.jsx

```diff
@@ -0,0 +1,12 @@
+import * as React from 'react';
+import Button from '@mui/material/Button';
+import Stack from '@mui/material/Stack';
+import { Link } from 'react-router-dom';
+
+export default function CreateProductButton(props) {
```

Comment:
This is ok, but in other cards, avoid creating a component only to wrap a button component.